style_guide.md

```diff
@@ -108,6 +106,16 @@ Don't tolerate warnings and don't suppress them due to laziness.
 
 Use the following source file layouts.
 
+Precede comments with a blank line.
```

Comment:
This is more accurately: the closing of a scope is followed by a blank line, I'll update.